0-common/scala-core/.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: scala
+scala:
+   - 2.10.6
+   - 2.11.8
+jdk:
+  - oraclejdk7
+  - openjdk7
```

Comment:
Not sure. @ninjabear mentioned there's no OpenJDK 8 in Travis CI. And seems this is true https://docs.travis-ci.com/user/languages/java, so I think we cannot.